content/docs/custom-plugins/configuring/index.md

```diff
@@ -6,35 +6,35 @@ description: How to configure a custom Backstage plugin to Roadie
 
 ## Introduction
 
-Roadie supports all standard Backstage frontend plugin types that can be built using instructions defined in [Backstage documentation][backstage-plugin-documentation]. Your plugins can be published via npm or yarn, like publishing a normal package. We will provide you functionality to automatically add custom plugins from repositories to your Roadie instance. 
+Roadie supports all standard Backstage frontend plugin types that can be built using instructions defined in [Backstage documentation][backstage-plugin-documentation]. Your plugins can be published via npm or yarn, like publishing a normal package. We will provide you functionality to automatically add custom plugins from repositories to your Roadie instance.
 
-If you you would like to use our private hosted repository for your Roadie plugins, complete with automatic update workflow, you can follow instructions on [setting up private secure Artifactory for Roadie plugins.](/docs/custom-plugins/artifactory/)
+If you want to have a private hosted repository for your Roadie plugins with automatic update workflow you can follow instructions on [setting up private secure Artifactory for Roadie plugins.](/docs/custom-plugins/artifactory/)
 
 ## Prerequisites
 
-Contact us to have the custom plugins feature enabled for your Roadie instance.
+Custom plugins need to be enabled for your Roadie instance. You can enable this by contacting us.
 
 ## Configuring a plugin for your Roadie instance
 
 ### Step 1. Create plugin definition in Roadie
 
-You can navigate to h<gatsbyhack>tt</gatsbyhack>ps://your-company.roadie.so/administration/custom-plugins where you will find a collection of cards detailing custom plugins configured for the Roadie instance. 
+You can navigate to h<gatsbyhack>tt</gatsbyhack>ps://your-company.roadie.so/administration/custom-plugins where you will find a collection of cards detailing custom plugins configured for the Roadie instance.
```

Comment:
Is the `<gatsbyhack>` there to prevent Gatsby rendering a link?

Reply:
yeah markdown doesn't have functionality to not automatically render a link-ish text as link so this will circumvent that.

Reply:
Nice trick!